web/packages/teleport/src/Login/Login.tsx

```diff
@@ -41,25 +42,35 @@ export function Login({
   isPasswordlessEnabled,
   primaryAuthType,
   privateKeyPolicyEnabled,
+  motd,
 }: State) {
+  const [showMOTD, setShowMOTD] = useState<boolean>(true);
+
+  const acknowledgeMOTD = () => {
+    setShowMOTD(false);
+  };
   return (
     <>
       <Logo src={logoSrc} />
-      <FormLogin
-        title={'Sign in to Teleport'}
-        authProviders={authProviders}
-        auth2faType={auth2faType}
-        preferredMfaType={preferredMfaType}
-        isLocalAuthEnabled={isLocalAuthEnabled}
-        onLoginWithSso={onLoginWithSso}
-        onLoginWithWebauthn={onLoginWithWebauthn}
-        onLogin={onLogin}
-        attempt={attempt}
-        clearAttempt={clearAttempt}
-        isPasswordlessEnabled={isPasswordlessEnabled}
-        primaryAuthType={primaryAuthType}
-        privateKeyPolicyEnabled={privateKeyPolicyEnabled}
-      />
+      {motd && showMOTD ? (
```

Comment:
add behavioral test?

Reply:
Added tests in https://github.com/gravitational/teleport/pull/27813/commits/36314beaa6c0cc0ff2c79b90bf2f0748697de19c